package.json

```diff
@@ -20,6 +20,10 @@
       {
         "command": "arduino-littlefs-upload.uploadLittleFS",
         "title": "Upload LittleFS to Pico/ESP8266/ESP32"
+      },
+      {
+        "command": "arduino-littlefs-upload.findPartitionFile",
+        "title": "Find Partition File"
```

Comment:
Just to be clear, in case other chips have this, can we call it `Find ESP32 Partition File` in the menu instead?

Also, in practical use, is this really necessary for users?  I think with your changes you're already checking and finding the "right" file.  And there's nothing the user can do with the printout of this command, right?

Reply:
You're right that it's not strictly necessary. The use case is to help locate the built-in preset partition files buried in the ESP32 Arduino IDE plugin installation folder, especially since the filenames often don't match the menu options. That, and it's confirmation that a `partitions.csv` is in the sketch folder, since the IDE alone seems to give no indication that one is present.

That said, the active partitions file can fairly easily be found manually, though I find the process cumbersome unless I already have the files open.

Now that I think more about it, the upload terminal log already tells the user which scheme (by the name of the CSV) is active. If you think that a command to print out the full path is bloat, I'll remove it since it's mostly just useful for newbs like me (got my first ESP32 threeish days ago :) ).

Reply:
Could you please remove it, then?  We do dump where we're looking for things when they try to build a FS anyway, like you said.  Thx!

Reply:
Hi @schmidtsamuelw and @earlephilhower ,

thank you for your efforts for this uploader.

I'm new to ESP Programming and the IDE, but stubled across exactly that problem: my sketch requires a non-default partition scheme, but the uploader does not honor the selected scheme. So I'm really in need of this fix.

I'm looking forward for the new release. Thank you very much!

Reply:
Hi, thanks for your patience - I've been busy away for my computer and found a bit of time to cut the code and readme line out via vscode.dev. With that said, I unfortunately won't back to my environment for a day and a half or so, so I'd appreciate if you could quickly package it and see if I didn't accidentally break something.